crates/tauri-runtime-wry/src/webview.rs

```diff
@@ -13,25 +13,17 @@ mod imp {
   pub type Webview = webkit2gtk::WebView;
 }
 
-#[cfg(target_os = "macos")]
+#[cfg(target_vendor = "apple")]
 mod imp {
-  use cocoa::base::id;
+  use std::ffi::c_void;
 
   pub struct Webview {
-    pub webview: id,
-    pub manager: id,
-    pub ns_window: id,
-  }
-}
-
-#[cfg(target_os = "ios")]
-mod imp {
-  use cocoa::base::id;
-
-  pub struct Webview {
-    pub webview: id,
-    pub manager: id,
-    pub view_controller: id,
+    pub webview: *mut c_void,
+    pub manager: *mut c_void,
+    #[cfg(target_os = "macos")]
+    pub ns_window: *mut c_void,
+    #[cfg(not(target_os = "macos"))]
```

Comment:
```suggestion
    #[cfg(target_os = "ios")]
```
just for consistency with the usage

Reply:
also makes it easier for us to locate it when we want to support visionOS